docs/en/connector-v2/sink/Doris.md

```diff
@@ -51,6 +51,12 @@ The internal implementation of Doris sink connector is cached and imported by st
 | custom_sql                     | String  | no       | -                            | When data_save_mode selects CUSTOM_PROCESSING, you should fill in the CUSTOM_SQL parameter. This parameter usually fills in a SQL that can be executed. SQL will be executed before synchronization tasks.                                                                             |
 | doris.config                   | map     | yes      | -                            | This option is used to support operations such as `insert`, `delete`, and `update` when automatically generate sql,and supported formats.                                                                                                                                              |
 
+### Note
```

Comment:
It's weird to put `Note` in Sink Options part. 
How about add new part named `Tuning Guide` before `Task Example` part?

Reply:
Ok